packages/status-page/src/pages/home/Home.svelte

```diff
@@ -167,9 +166,9 @@
     {
       statusFunc: getPeerCount,
       watchStatusFunc: null,
-      provider: l2BootnodeProvider,
+      provider: l2Provider,
       contractAddress: "",
-      header: "Peers",
+      header: "Peers (Taiko Node)",
```

Comment:
hmm, since we didn't increase the RPC node's `maxPeer` config (and i believe we shouldn't...), this number will be relatively small (~20),so i dont think showing this number will make much sense?

Reply:
It was our best guess attempt at a number of ndoes, but I can remove it.

Reply:
I think Ben was saying it was a community requested feature on the discord team channel

Reply:
We can have it for now and optimize what shall be displayed later.